ddl/placement/utils.go

```diff
@@ -20,6 +20,7 @@ import (
 	"strings"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/store/tikv/oracle"
```

Comment:
ddl/placement only focuses placement rules utilities. Importing transaction utility functions here might be not suitable. Any more suitable places to place this function?